src/DesktopHeader.jsx

```diff
@@ -54,6 +54,24 @@ class DesktopHeader extends React.Component {
     });
   }
 
+  // Renders an optional App Menu for
+  renderAppMenu() {
+    const { appMenu } = this.props;
+    const { content, href, menuItems } = appMenu;
+    return (
+      <Menu transitionClassName="menu-dropdown" transitionTimeout={250}>
+        <MenuTrigger tag="a" className="nav-link d-inline-flex align-items-center" href={href}>
```

Comment:
I think we can drop the `href`.  Having a link that also works as a dropdown can be midly confusing.

Reply:
href removed!